core/controllers/galleries.py

```diff
@@ -112,7 +112,7 @@ def get(self):
 
         self.values.update({
             'activity_summary_dicts_by_category': (
-                activity_summary_dicts_by_category),
+                summary_dicts_by_category),
```

Comment:
I think we should actually add a new key here: 'featured_activity_summary_dicts' -- and return the new item separately.